test/drbgtest.c

```diff
@@ -414,7 +418,7 @@ static int error_check(DRBG_SELFTEST_DATA *td)
      * failure.
      */
     t.entropylen = 0;
-    if (TEST_false(RAND_DRBG_generate(drbg, buff, td->exlen, 1,
+    if (!TEST_false(RAND_DRBG_generate(drbg, buff, td->exlen, 1,
```

Comment:
This is an expected fail. I must say these !TEST_something are really confusing and hard to parse.

Reply:
> This is an expected fail. 

Correct. This error seems to have slipped in already when this module was added.

>  I must say these !TEST_something are really confusing and hard to parse.

I fully agree. I had a hard time too reading these conditions at the beginning. My strategy is to just ignore the surrounding `if (! .... )`, because it simply means "break on failure of the test condition", so stripping it leaves only the test condition: Instead of

    if (!TEST_false(RAND_DRBG_generate(...)))

I just read

    TEST_false(RAND_DRBG_generate(...))